tests/integration/tests/cluster/test-cluster-role-bindings.spec.js

```diff
@@ -106,10 +106,9 @@ context('Test Cluster Role Bindings', () => {
   });
 
   it('Delete Cluster Role Binding', () => {
-    cy.wait(500)
-      .get('ui5-input[id="search-input"]:visible')
-      .find('input')
-      .type(CRB_NAME);
+    cy.wait(500);
+
+    cy.typeInSearch(CRB_NAME);
```

Comment:
Didn't test it, should work like this
```suggestion
    cy.wait(500).typeInSearch(CRB_NAME);
```